NETCORE/.vsts/linux-build.yml

```diff
@@ -29,10 +32,21 @@ steps:
     projects: "NETCORE/test/**/TestApp30.Tests30.csproj"
     arguments: "--configuration Release -l trx"
 
-- task: DotNetCoreInstaller@2
-  displayName: install dotnet core 2.2.104
+- task: DotNetCoreCLI@1
+  displayName: Unit Tests for AspNetCore 3.1
+  continueOnError: true
+  inputs:
+    command: "test"
+    projects: "NETCORE/test/**/*AspNetCore.Tests.csproj"
+    arguments: "--configuration Release --framework netcoreapp3.1 -l trx"
```

Comment:
these tests aren't being run. We need to run this command locally and confirm that it's working.

Reply:
ok! just changed to full name and it worked!